css/ampshared.css

```diff
@@ -264,14 +264,19 @@ i-amphtml-sizer {
  * [width][height][sizes] and [width][height][heights] imply layout=responsive
  * before the element has resolved.
  */
-.i-amphtml-notbuilt:not(.i-amphtml-layout-container) > * ,
+.i-amphtml-notbuilt:not(.i-amphtml-layout-container) > *,
 [layout]:not([layout=container]):not(.i-amphtml-element) > *,
 [width][height][sizes]:not([layout]):not(.i-amphtml-element) > *,
 [width][height][heights]:not([layout]):not(.i-amphtml-element) > *
 {
   display: none;
 }
 
+amp-img:not(.i-amphtml-element)[i-amphtml-ssr] > img.i-amphtml-fill-content
```

Comment:
Looks good, but please add a comment explaining this and highlight that it needs to be reviewed when more elements are SSRd.